zk_toolbox/crates/config/src/ecosystem.rs

```diff
@@ -25,6 +25,7 @@ struct EcosystemConfigInternal {
     pub name: String,
     pub l1_network: L1Network,
     pub link_to_code: PathBuf,
+    pub bellman_cuda_dir: PathBuf,
```

Comment:
I assume it's suppose to be optional, because we don't need always run and install provers